recipes/pymvpa2/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set name = "pymvpa2" %}
+{% set version = "2.6.1" %}
+{% set sha256 = "206fa302fc6292831c50d6ffe9b1a95fc048380b0701c140eb7300dcd017052a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py>3]
```

Comment:
So there is no version of `python` greater than 3 yet. What constraint were you meaning to place here?

Reply:
Isn't `3.6 > 3`, ie this is the same as `py3k`?

Reply:
Apparently not as all builds were skipped in the feedstock. So my understanding is it thinks `py>3` means that it should be `python >=4`.

Note that this logic is just propagated up from `conda-build`. We only use it to determine which builds should or shouldn't be allowed at the `conda-build-all` level.

Reply:
I think I did have the same understanding/hope that any 3.x would be greater than 3.  We do not support python3 yet fully unfortunately.  It might build/install and may be even pass the tests, but ideally shouldn't be provided for python3 at least for now.  I guess I will correct it later straight in the feedstock to avoid failing attempts to build for python3

Reply:
So an attempt to fix this is in PR ( https://github.com/conda-forge/pymvpa2-feedstock/pull/1 ). Would be great if you guys took a look.